Resources/Prototypes/_SpaceCats/Entities/Structures/Machines/Computers/book_printer.yml

```
@@ -0,0 +1,119 @@
+    sound:
+      path: /Audio/Ambience/Objects/engine_hum.ogg
+
+- type: entity
+  id: BookPrinterFilled
+  parent: BookPrinter
+  suffix: Filled
+  components:
+  - type: ContainerFill
+    containers:
+      cartridgeSlot:
+      - BookPrinterInkCartridge
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)